wrappers/PhyML/README.md

```diff
@@ -0,0 +1,70 @@
+# **PhyML Galaxy Wrapper** <br>
+This repository contains a Galaxy tool wrapper for the PhyML program, which estimates the maximum likelihood phylogenies from alignments of nucleotide or amino acid sequences. <br>
+PhyML has been developed within the PhyML Development Team at ATGC Montpellier Bioinformatics platform. (*http://www.atgc-montpellier.fr/phyml/*)
+
+The reference for PhyML is: <br>
+	<pre>*New Alogirthms and Methods to Estimate Maximum-Likelihood Phylogenies: Assessing the Performance of PhyML* 
+Guindon S., Dufayard J.F., Lefort V., Anisimova M., Hordijk W., & Gascuel O. (2010). 3.0. 
+System Biology, 59(3):307-21. </pre>
+
+The PhyML Tool Suite repository: <br>
+<pre>&#9;* Provides PhyML wrapper for the PhyML tool 
+&#9;* Downloads and installs PhyML on the Linux operating system  </pre>
```

Comment:
Try not to use any special characters or codes, since this file could be viewed as a simple text file.